File: doc/snippets/whatsapp/file/retrieve/snippet.py
import requests
import json

url = 'https://api.wababox.net/waba/1.1/json/<WABAID>/file/<PHONEID>'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    'file_id': '1234567890'
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())

File: doc/snippets/whatsapp/file/retrieve/snippet.js
const url = 'https://api.wababox.net/waba/1.1/json/<WABAID>/file/<PHONEID>';
const headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
};
const data = {
    file_id: '1234567890',
};

fetch(url, {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(data)
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));
